Src/Support/GoogleApis.Auth/OAuth2/Requests/GoogleAuthorizationCodeRequestUrl.cs

```diff
@@ -60,6 +61,14 @@ public class GoogleAuthorizationCodeRequestUrl : AuthorizationCodeRequestUrl
         public string IncludeGrantedScopes { get; set; }
 
         /// <summary>
+        /// Gets or sets a collection of user defined query parameters to facilitate any not explicitly supported
+        /// by the library which will be included in the resultant authentication URL.
+        /// </summary>
+        [Google.Apis.Util.RequestParameterAttribute("user_defined_query_parameters",

```

Comment:
Like I suggested in the closed PR. I would add a new RequestParameterAttribute constructor that gets only the type (no name).

The name isn't going to be used, so it will make it clearer to not include it here.
Does it make sense?